authentik/admin/api/system.py

```diff
@@ -25,11 +27,13 @@ class RuntimeDict(TypedDict):
     """Runtime information"""
 
     python_version: str
-    gunicorn_version: str
     environment: str
     architecture: str
     platform: str
     uname: str
+    openssl_version: str
+    openssl_fips_mode: bool
+    authentik_version: str
```

Comment:
TODO: also do this for outposts

Reply:
added with [`49e836f` (#9683)](https://github.com/goauthentik/authentik/pull/9683/commits/49e836f60bf607f16157015020cbea947d0a00a4)